charts/flux2-sync/README.md

```diff
@@ -14,36 +14,40 @@ This helm chart is maintain and released by the fluxcd-community on a best effor
 
 | Key | Type | Default | Description |
 |-----|------|---------|-------------|
-| gitRepository.spec.gitImplementation | string | `""` | (Optional) Determines which git client library to use. Defaults to go-git, valid values are (‘go-git’, ‘libgit2’). |
-| gitRepository.spec.ignore | string | `""` | (Optional) Ignore overrides the set of excluded patterns in the .sourceignore format (which is the same as .gitignore). If not provided, a default will be used, consult the documentation for your version to find out what those are. |
-| gitRepository.spec.include | list | `[]` | (Optional) Extra git repositories to map into the repository |
+| cli.affinity | object | `{}` |  |
+| cli.image | string | `"ghcr.io/fluxcd/flux-cli"` |  |
+| cli.nodeSelector | object | `{}` |  |
+| cli.tag | string | `"v0.30.2"` |  |
+| cli.tolerations | list | `[]` |  |
+| gitRepository.spec.gitImplementation | Optional | `""` | Determines which git client library to use. Defaults to go-git, valid values are (‘go-git’, ‘libgit2’). |
```

Comment:
There appears to be some differences in the table columns -- type is missing from optional values -- did helmdocs do this?

Reply:
It did, I'll look into why that is. Not sure if its my helmdocs version?

Reply:
Yep seems to be a new feature apart of `helm-docs@1.10.0`: https://github.com/norwoodj/helm-docs/commit/d2cc641f41278f50b5d23a54a7c889e4427f9417

> - `Type`: the type of the value of the key in `values.yaml`. Can be automatically inferred from YAML structure, or annotated using `# -- (mytype)` where `mytype` can be any string that you refer as the type of the value.

Which is exactly how the values comments currently indicate optional: https://github.com/fluxcd-community/helm-charts/blob/main/charts/flux2-sync/values.yaml#L30

So I've changed the comments from `(Optional)` to `_Optional_` instead so its italicized. If you'd rather me leave the comments / docs as is and downgrade my `helm-docs` thats fine too.